### ISSUE TITLE
real world breakage

### ISSUE THREAD

**Issue by samccone** [PolymerLabs/polydev#21]
Reported here
https://github.com/PolymerElements/iron-overlay-behavior/issues/39#issuecomment-144788553

**Comment by samccone** [comment]
Another mention here:

https://github.com/PolymerElements/iron-overlay-behavior/commit/ad1cda00708d1a79539b7ef511454d576ed779f6

**Comment by 01es** [comment]
Just would like to second the reported problem and provide a stack trace. 

``` JavaScript
Uncaught TypeError: Cannot read property 'toString' of null
    obj.addEventListener @ polydev/vendor/zone-microtask.js:973
    Gestures.add @ polymer.html:774
    Polymer.IronOverlayBehaviorImpl._toggleListener @ iron-overlay-behavior.html:268
    (anonymous function) @ iron-overlay-behavior.html:286
    (anonymous function) @ polymer.html:1355
    arguments.(anonymous function) @ polydev/vendor/zone-microtask.js:406
    (anonymous function) @ polydev/vendor/zone-microtask.js:98
    run @ polydev/vendor/zone-microtask.js:118
    zoneBoundFn @ polydev/vendor/zone-microtask.js:91
```

**Comment by ebidel** [comment]
Did you install the extension on a fresh profile?

**Comment by 01es** [comment]
I'm not certain what does the "fresh profile" mean... If this means a new/reset Chrome browser user profile then no. However, the reported problem was experienced with a freshly installed [polydev extension](https://chrome.google.com/webstore/detail/polymer-devtools-extensio/mmpfaamodhhlbadloaibpocmcomledcg) version 0.0.6. There was no earlier version of this extension installed.

**Comment by ebidel** [comment]
See https://github.com/PolymerLabs/polydev/issues/26#issuecomment-158975027. We recommend using this extension in a fresh chrome profile b/c it is known to be intrusive.

**Comment by samccone** [comment]
#30, #28

**Comment by 01es** [comment]
Chrome Canary was installed. New profile added. Polydev installed from chrome web store. However, when running our Polymer based application the following errors occur upon opening an overlay. Any suggestions?

``` JavaScript
polydev/vendor/zone-microtask.js:973 Uncaught TypeError: Cannot read property 'toString' of null
obj.addEventListener @ polydev/vendor/zone-microtask.js:973
Gestures.add @ polymer.html:774
Polymer.IronOverlayBehaviorImpl._toggleListener @ iron-overlay-behavior.html:268
(anonymous function) @ iron-overlay-behavior.html:286
(anonymous function) @ polymer.html:1355
arguments.(anonymous function) @ polydev/vendor/zone-microtask.js:406
(anonymous function) @ polydev/vendor/zone-microtask.js:98
run @ polydev/vendor/zone-microtask.js:118
zoneBoundFn @ polydev/vendor/zone-microtask.js:91

polydev/vendor/zone-microtask.js:995 Uncaught TypeError: Cannot read property '_bound' of null
obj.removeEventListener @ polydev/vendor/zone-microtask.js:995
Gestures.remove @ polymer.html:798
Polymer.IronOverlayBehaviorImpl._toggleListener @ iron-overlay-behavior.html:274
(anonymous function) @ iron-overlay-behavior.html:286
(anonymous function) @ polymer.html:1355
arguments.(anonymous function) @ polydev/vendor/zone-microtask.js:406
(anonymous function) @ polydev/vendor/zone-microtask.js:98
run @ polydev/vendor/zone-microtask.js:118zoneBoundFn @ polydev/vendor/zone-microtask.js:91
```

**Comment by samccone** [comment]
@01es it is a known issue with this project that it does not work for a subset of polymer projects, installing on a clean profile will only make it so your normal internet browsing experience is not silently broken.

As for a solution the bug most likely can be found / fixed here
https://github.com/PolymerLabs/element-zones

**Comment by fozziethebeat** [comment]
Running PolymDev on https://translate.google.com/community currently breaks the site completely and prevents it from loading.  It previously worked.

Has this kept up to date with Chrome or Polymer?  Is this going to be updated anytime soon so that it doesn't require more steps for setup?

**Comment by rictic** [comment]
This should be fixed in v0.0.7 which was just merged in #35 and published. Please file a new issue if you see any more problems like this.
